scripts/eurostat/gdp/preprocess_data.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Defines functions to preprocess Eurostat data and put it in long-form CSV format.
+
+Author: jinpz
+
+    Usage:
+
+    python3 preprocess_data.py (--input_file) (--output_file) (--sep)
+"""
+import pandas as pd
+import argparse
+import gzip
+import shutil
+
+# TODO (jinpz): Add comments below to make this script easier to read.
```

Comment:
Please add
\# TODO(fpernice): Replace with the final submitted code for Jin Zhou's Eurostat import code.

Reply:
Done.